ando/checker.py

```diff
@@ -1,28 +1,25 @@
-#!/usr/bin/python
+# File: checker.py
+# Project: ando
+# File Created: Tuesday, 30th June 2020 10:50:05 am
+# Author: garcia.j (Jeremy.garcia@univ-amu.fr)
+# -----
+# Last Modified: Thursday, 2nd July 2020 1:29:58 pm
+# Modified By: garcia.j (Jeremy.garcia@univ-amu.fr)
+# -----
+# Copyright - 2020 MIT, Institue de neurosciences de la Timone
+
```

Comment:
it seems you've added this everywhere? It looks like it's automatic... Is this intended?
Anyhow, I have three remarks:
- there's a typo at InstituE, and add a capital letter at Neurosciences
- you should add CNRS, next to the copyright line
- about the MIT licence... we should probably discuss this (but I never know which license to choose)

Reply:
Done